predicators/envs/behavior.py

```diff
@@ -508,7 +508,12 @@ def render_state(self,
                         "behavior_mode in settings.py instead")
 
     def _get_task_relevant_objects(self) -> List["ArticulatedObject"]:
-        return list(self.igibson_behavior_env.task.object_scope.values())
+        additional_objs = [
+            obj for obj in self.igibson_behavior_env.scene.get_objects()
+            if "board_game" in obj.name
+        ]
```

Comment:
leave a comment explaining this so that we don't forget in the future/new ppl understand what's going on here

Reply:
Good point.